distributed/scheduler.py

```diff
@@ -5952,7 +5952,7 @@ def profile_to_figure(state):
 
         # Task stream
         task_stream = self.get_task_stream(start=start)
-        total_tasks = len(task_stream)
+        total_tasks = self.n_tasks
```

Comment:
Ideally the performance report would capture only the number of tasks during execution, rather than for the lifetime of the scheduler.


So for example in the following situation we would want only the number of tasks from `y.compute()` not also `x.compute()`

```python
x.compute()

with performance_report():
    y.compute()
```

Reply:
good point!